hardware/computemodule/designfiles.md

```diff
@@ -1,6 +1,10 @@
 # Compute Module IO board and camera/display adapter board design data
 
-Design data for the Compute Module IO board (CMIO)
+Design data for the Compute Module 4 IO board can be found in its datasheet
+
+- [Compute Module 4 IO Board datasheet](http://datasheets.raspberrypi.org/cm4io/cm4io-datasheet.pdf)
```

Comment:
Might want to add a link to the KiCad file here?

http://datasheets.raspberrypi.org/cm4io/CM4IO-KiCAD.zip

this is the actual design files for the board.